.github/workflows/ci.yaml

```diff
@@ -64,6 +65,7 @@ jobs:
     - uses: actions/setup-go@v3
       with:
         go-version: v1.18
+        cache: true
```

Comment:
https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows says

> Use cache-dependency-path input for cases when multiple dependency files are used, or they are located in different subdirectories.

I don't know if we can (ab)use that to get `ko`'s installation dir or not.